core/ide/che-core-ide-stacks/src/main/resources/predefined-stacks.json

```diff
@@ -696,5 +696,87 @@
       "name": "type-java.svg",
       "mediaType": "image/svg+xml"
     }
-  }
+  },
+  {
+    "id": "tomee-default",
+    "creator": "tomitribe",
+    "name": "TomEE",
+    "description": "Default Java Stack with JDK 8, Maven and Apache TomEE 1.7.3.",
+    "scope": "general",
```

Comment:
it should be "advanced"  instead of "general" where we only have one per technology : java, php, node, etc